### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/08/17/day16.html

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#26] [Comments]
# https://mnfienen.github.io/talulat/2023/08/17/day16.html

Day 16 - Back on the south side of the St. Lawrence

https://mnfienen.github.io/talulat/2023/08/17/day16.html

**Comment by aleaf** [comment]
Looks like a great campsite!

**Comment by mnfienen** [comment]
yeah - was really spectacular! Only issue was the mosquitos really loved it too :)
